### PR TITLE
Implemented throw error if there no base templates

### LINKED ISSUE #83: Throw error if there is no base templates
If you do not add base teamples — assembly will be successful, but if you call BEMHTML.apply(bemjson) you will see strange error:

```
Error: [object global]
      at $e (bundle/bundle.bemhtml.js:27:15)
      at $2 (bundle/bundle.bemhtml.js:23:20)
      at $2 (bundle/bundle.bemhtml.js:17:27)
      at applyc (bundle/bundle.bemhtml.js:37:20)
      at Array.apply (bundle/bundle.bemhtml.js:32:16)
      at bundle/bundle.bemhtml.js:79:25
      at Object.<anonymous> (bundle/bundle.bemhtml.js:82:7)
      at test/techs/bemhtml/bemhtml.test.js:263:21
```

### PR DESCRIPTION
**Pull request by blond**
Resolved #83

### CODE REVIEW

**Comment by tormozz48** [comment]
:ok:

File: test/techs/bemhtml/bemhtml.test.js [hunk 1]
```diff
@@ -29,6 +29,34 @@ describe('bemhtml', function () {
         return assert(bemjson, html, templates);
     });
 
+    describe('not base templates', function () {
```
Comment (tadatuta) [inline test/techs/bemhtml/bemhtml.test.js]: not base templates -> no base templates

File: test/techs/bemhtml/bemhtml.test.js [hunk 2]
```diff
@@ -29,6 +29,34 @@ describe('bemhtml', function () {
...
+        it('should throw valid error if base template is missed (for development mode)', function () {
```
Comment (tadatuta) [inline test/techs/bemhtml/bemhtml.test.js]: is missing

File: test/techs/bemhtml/bemhtml.test.js [hunk 3]
```diff
@@ -240,9 +268,13 @@ function build(templates, options) {
         },
         bundle, fileList;
 
-    templates && templates.forEach(function (item, i) {
-        scheme.blocks['block-' + i + '.bemhtml'] = item;
-    });
+    if (Array.isArray(templates)) {
+        templates && templates.forEach(function (item, i) {
```
Comment (tadatuta) [inline test/techs/bemhtml/bemhtml.test.js]: no need for `templates &&` part

File: test/techs/bemtree.test.js [hunk 4]
```diff
@@ -85,6 +85,38 @@ describe('bemtree', function () {
             });
     });
 
+    describe('not base templates', function () {
```
Comment (tadatuta) [inline test/techs/bemtree.test.js]: not -> no

File: test/techs/bemtree.test.js [hunk 5]
```diff
@@ -85,6 +85,38 @@ describe('bemtree', function () {
...
+        it('should throw valid error if base template is missed (for development mode)', function () {
```
Comment (tadatuta) [inline test/techs/bemtree.test.js]: missing

File: test/techs/bemtree.test.js [hunk 6]
```diff
@@ -85,6 +85,38 @@ describe('bemtree', function () {
...
+             return build(blocks, { devMode: true })
+                 .spread(function (res) {
+                     return res.BEMTREE.apply(data);
+                 })
+                 .fail(function (error) {
+                     error.message.must.be.equal('Seems like you have no base templates from i-bem.bemtree.xjst');
+                 });
+         });
+
+         it('should throw valid error if base template is missed (for production mode)', function () {
```
Comment (tadatuta) [inline test/techs/bemtree.test.js]: missing

**Pull request merged**